core/comment.js

```diff
@@ -10,43 +10,44 @@
  */
 'use strict';
 
-goog.provide('Blockly.Comment');
-
-goog.require('Blockly.browserEvents');
-goog.require('Blockly.Bubble');
-goog.require('Blockly.Css');
-goog.require('Blockly.Events');
+goog.module('Blockly.Comment');
+goog.module.declareLegacyNamespace();
+
+const Block = goog.requireType('Blockly.Block');
+const BlockSvg = goog.requireType('Blockly.BlockSvg');
+const browserEvents = goog.require('Blockly.browserEvents');
```

Comment:
I think this is something that should be destructured.
```
const {conditionalBind, data, unbind} = goog.require('Blockly.browserEvents');
```

Same for `dom`, `userAgent`, and `utilsObject`.

Reply:
Done!